packages/core/src/mirrorworld.ts

```diff
@@ -1198,12 +1216,20 @@ export class MirrorWorld {
     const Asset = Object.freeze({
       /** Purchase NFT on Polygon Marketplace Address */
       buyNFT: this.buyEVMNFT.bind(this),
-      /** Purchase NFT on Polygon Marketplace Address */
+      /** List NFT on Polygon Marketplace Address */
       listNFT: this.listEVMNFT.bind(this),
       /** Cancel NFT Listing on Polygon Marketplace Address */
       cancelListing: this.cancelEVMNFTListing.bind(this),
       /**  Transfer Polygon NFT to another address */
       transferNFT: this.transferEVMNFT.bind(this),
+      /** Purchase NFT on Ethereum Marketplace Address */
+      buyNFTTransaction: this.buyEVMNFTTransaction.bind(this),
+      /** List NFT on Ethereum Marketplace Address with External Wallet */
+      listNFTTransaction: this.listEVMNFTTransaction.bind(this),
+      /** Cancel NFT Listing on Ethereum Marketplace Address with External Wallet */
+      cancelListingTransaction: this.cancelEVMNFTListingTransaction.bind(this),
+      /**  Transfer Ethereum NFT to another address with External Wallet */
+      transferNFTTransaction: this.transferNFTTransaction.bind(this),
```

Comment:
LEt's improve comments to match the one above.

Reply:
updated it 👍🏻